### PR TITLE
Circuit Imprinters

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
Adds circuit imprinters, which now are required for all machine circuit boards. Adjusts circuit board recipes to be more like their most common implementation. Removes circuitboards from the protolathe.

**Screenshots**

**Changelog**

:cl: Rane
- add: Added Circuit Imprinters. These machines are used to print all machine circuit boards, instead of the protolathe.
- tweak: Circuit board recipes tweaked. They now use a lot of glass, a bit of metal, and a little bit of gold for the rarest ones.

### CODE REVIEW

**Comment by Lamrr** [comment]
Would it be possible for it to require sulphuric acid like ones in SS13 do? Feel like it make sense + additional cost + science leaves the department for something at least once.

**Comment by Elijahrane** [comment]
> Would it be possible for it to require sulphuric acid like ones in SS13 do? Feel like it make sense + additional cost + science leaves the department for something at least once.

1. Basing it off /tg/ ones which don't require that
2. WYCI

File: Resources/Prototypes/Catalog/Fills/Lockers/heads.yml [hunk 1]
```diff
@@ -156,6 +156,8 @@
         prob: 1
       - id: ProtolatheMachineCircuitboard
         prob: 1
+      - id: CircuitImprinterMachineCircuitboard
+        prov: 1
```
Comment (metalgearsloth) [inline Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:160]:
```suggestion
```

Don't need and it's misspelt.

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml [hunk 2]
```diff
@@ -216,3 +203,41 @@
...
+    - state: icon
+      map: ["enum.ProtolatheVisualLayers.Base"]
+    - state: unlit
+      shader: unshaded
+      map: ["enum.ProtolatheVisualLayers.BaseUnlit"]
+    - state: icon
+      map: ["enum.ProtolatheVisualLayers.AnimationLayer"]
+    - state: panel
+      map: ["enum.WiresVisualLayers.MaintenancePanel"]
+  - type: ProtolatheDatabase
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Structures/Machines/lathe.yml:225]: Parent should be netsync false unless required (one idea we floated a while back was making sprites client-only but too much stuff still sets sprites on server).

**Comment by Elijahrane** [comment]
Ping me when you want to merge and I'll map them in again, I'm not dealing with the cursed map merge thing

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**